packages/wonder-blocks-modal/util/util.js

```diff
@@ -1,21 +0,0 @@
-// @flow
```

Comment:
We're not using anything from this file anymore so it's safe to remove it!

Reply:
Oh, how are we doing a11y now? Are we assuming that there will only be one modal component instance per page? If so, how does this interact with cases where a page uses a modal layout inline, but might also render a modal on top of it?

Reply:
We can tackle this in a follow up PR.  I'll add an issue in JIRA.